ios/README.md

```
@@ -60,15 +60,13 @@
 ## Localization
 
-### Twine
-
-- All strings in the application are localized and shared with the Android team
-  via [Twine](https://github.com/scelis/twine)
-- Strings are stored in the `twine/strings.txt` file
-- TwinePlugin then generates appropriate `Localizable.strings` files from the
-  mentioned `strings.txt` file
-- When modifying `strings.txt` it is required to comply with the specified syntax and to pull/push
-  all the changes frequently
+#### Sentiary
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)